Makefile

```diff
@@ -24,11 +23,17 @@ require-goversion:
 		fi; \
 	fi;
 
-build: require-goversion
-	$(BUILD_FLAGS) go build -ldflags="-X ${PKG}.GolangVersion=${GOLANG_VERSION} ${GOVVV_FLAGS}" .
+require-govvv:
+	$(eval govvv_loc := $(shell which govvv))
+	@if [ "$(govvv_loc)" == "" ]; then go get github.com/ahmetb/govvv; fi;
```

Comment:
`go get` will download the source code, but won't build govvv and put it onto the PATH, it needs `go install`.

Reply:
very good point, just force-pushed this change